blockbase/inc/customizer/wp-customize-colors.php

```diff
@@ -121,21 +131,45 @@ function register_color_controls( $wp_customize, $palette ) {
 		}
 	}
 
+	function register_duotone_controls( $wp_customize ) {
+		$wp_customize->add_setting(
+			'duotone_control',
+			array(
+				'default'           => true,
+				'capability'        => 'edit_theme_options',
+				'sanitize_callback' => array( __CLASS__, 'sanitize_boolean' ),
```

Comment:
Can we change this to https://developer.wordpress.org/reference/functions/rest_sanitize_boolean/ ?